src/views/registration/button-cart.php

```diff
@@ -16,7 +16,12 @@
 $provider     = $this->get( 'provider' );
 $cart_url     = $this->get_cart_url( $provider );
 $provider_obj = $this->get_cart_provider( $provider );
-$checkout_url = $provider_obj->get_checkout_url();
+
+if ( method_exists( $provider_obj, "get_cart_provider" ) ) {
```

Comment:
:no_entry_sign: String "get_cart_provider" does not require double quotes; use single quotes instead

<sub>hash: 5bf8d95ad6b4bcd28b159fa025106c02</sub>

Reply:
```suggestion
if ( method_exists( $provider_obj, 'get_cart_provider' ) ) {
```